config/database.yml

```diff
@@ -15,6 +15,9 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *defaults
+  username: <%= ENV['DATABASE_USER'] %>
+  password: <%= ENV['DATABASE_PASSWORD'] %>
+  host: <%= ENV['DATABASE_URL'] || '127.0.0.1' %>
```

Comment:
I don't think code changes to `config/database.yml` are needed. `<<: *defaults` should already be doing what is added here.

Reply:
👍🏽